### PR TITLE
Delete deprecated files.

### PR DESCRIPTION
**Pull request by dcr-stripe**
r? @kamil-stripe 

## Changelog
* Delete the `CashBalance.retrieve` instance method and rename the `CashBalance.retrieveCashBalance` static method to `CashBalance.retrieve`.
  * Previously if you had:
     ```
     CashBalance.retrieveCashBalance(customerId)
     ```
     this would become:
     ```
     CashBalance.retrieve(customerId)
     ```
* Delete `treasury.InitiatingPaymentMethodDetails` and `treasury.BillingDetails` top-level classes. These were unused and each treasury class has its own version of it.
* Delete `IssuerFraudRecord` resource. This resource is obsolete and has been superseded by `EarlyFraudWarning`.

### CODE REVIEW

File: src/main/java/com/stripe/model/CashBalance.java [hunk 1]
```diff
@@ -49,54 +48,19 @@ public class CashBalance extends ApiResource {
   BalanceSettings settings;
 
   /** Retrieves a customer’s cash balance. */
-  public static CashBalance retrieveCashBalance(String customer) throws StripeException {
```
Comment (kamil-stripe) [inline src/main/java/com/stripe/model/CashBalance.java:52]: This seems to not be a straight-forward removal

**Comment by kamil-stripe** [comment]
> Delete CashBalance.retrieve instance method. The CashBalance.**retrieve** static method is still available and should be used instead.

In the changelog.

File: src/main/java/com/stripe/model/CashBalance.java (hunk 1 above)
Comment (kamil-stripe) [reply src/main/java/com/stripe/model/CashBalance.java:52]: I see the description. Please update the PR's title :)

**Review by kamil-stripe — APPROVED** [review]

File: src/main/java/com/stripe/model/CashBalance.java (hunk 1 above)
Comment (dcr-stripe) [reply src/main/java/com/stripe/model/CashBalance.java:52]: Good catch! Updated.

**Pull request merged**